examples/single_model_scripts/run_swag.py

```diff
@@ -428,10 +428,10 @@ def main():
             warmup_linear = WarmupLinearSchedule(warmup=args.warmup_proportion,
                                                  t_total=num_train_optimization_steps)
         else:
-            optimizer = BertAdam(optimizer_grouped_parameters,
-                                 lr=args.learning_rate,
-                                 warmup=args.warmup_proportion,
-                                 t_total=num_train_optimization_steps)
+            optimizer = AdamW(optimizer_grouped_parameters, lr=args.learning_rate)
+            scheduler = WarmupLinearSchedule(optimizer,
```

Comment:
The scheduler should be created also if we are in fp16.
The fp16 optimizer should now be created like in the `run_glue` example where there is no distinction between fp16 and normal operation.